test_programs/cl4/1.lox

```diff
@@ -0,0 +1,20 @@
+---
+expected_error_type: none
+---
+class Robot {
+  beep() {
```

Comment:
Do one test that only has methods without arguments, and then include arguments from test 2 onwards